# Translations template for keystone.
# Copyright (C) 2013 ORGANIZATION
# This file is distributed under the same license as the keystone project.
# 
# Translators:
# FIRST AUTHOR <EMAIL@ADDRESS>, 2012.
#   <kelemeng@gnome.hu>, 2012.
msgid ""
msgstr ""
"Project-Id-Version: Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2013-01-01 00:00+0000\n"
"PO-Revision-Date: 2012-12-15 14:14+0000\n"
"Last-Translator: kelemeng <kelemeng@gnome.hu>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: keystone/clean.py:23
#, python-format
msgid "%s cannot be empty."
msgstr "%s nem lehet üres"

#: keystone/clean.py:25
#, python-format
msgid "%(property_name)s cannot be less than %(min_length)s characters."
msgstr "%(property_name)s nem lehet kevesebb, mint %(min_length)s karakter."

#: keystone/clean.py:29
#, python-format
msgid ""
"%(property_name)s should not be greater than %(max_length)s characters."
msgstr "%(property_name)s nem lehet több, mint %(max_length)s karakter."

#: keystone/clean.py:36
#, python-format
msgid "%(property_name)s is not a%(display_expected_type)s"
msgstr "%(property_name)s nem %(display_expected_type)s"

#: keystone/config.py:44
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr "Nem található a megadott naplózásbeállító fájl: %s"

#: keystone/config.py:62
msgid "Invalid syslog facility"
msgstr "Érvénytelen rendszernapló szolgáltatás"

#: keystone/test.py:103
#, python-format
msgid "Failed to checkout %s"
msgstr "Nem sikerült %s kiiktatása"

#: keystone/catalog/core.py:38
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr "Rosszul formázott végpont: %(url)s - ismeretlen kulcs: %(keyerror)s"

#: keystone/catalog/core.py:43
#, python-format
msgid ""
"Malformed endpoint %(url)s - unknown key %(keyerror)s(are you missing "
"brackets ?)"
msgstr "Rosszul formázott végpont: %(url)s - ismeretlen kulcs: %(keyerror)s (hiányoznak a zárójelek?)"

#: keystone/catalog/core.py:49
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format                   (are you missing"
" a type notifier ?)"
msgstr "Rosszul formázott végpont: %s hiányos formátum (hiányzik egy típusjelölő?)"

#: keystone/catalog/backends/templated.py:109
#, python-format
msgid "Unable to open template file %s"
msgstr "Nem nyitható meg a sablonfájl: %s"

#: keystone/common/bufferedhttp.py:102
#, python-format
msgid ""
"HTTP PERF: %(time).5f seconds to %(method)s %(host)s:%(port)s %(path)s)"
msgstr "HTTP PERF: %(time).5f másodperc eddig: %(method)s %(host)s:%(port)s %(path)s)"

#: keystone/common/cms.py:44
#, python-format
msgid "Verify error: %s"
msgstr "Ellenőrzési hiba: %s"

#: keystone/common/cms.py:134
#, python-format
msgid "Signing error: %s"
msgstr "Aláírási hiba: %s"

#: keystone/common/utils.py:124
msgid "using _calc_signature_2"
msgstr "a _calc_signature_2 használata"

#: keystone/common/utils.py:140
#, python-format
msgid "query string: %s"
msgstr "lekérdezési karakterlánc: %s"

#: keystone/common/utils.py:142
#, python-format
msgid "string_to_sign: %s"
msgstr "string_to_sign: %s"

#: keystone/common/utils.py:145
#, python-format
msgid "len(b64)=%d"
msgstr "len(b64)=%d"

#: keystone/common/utils.py:146
#, python-format
msgid "base64 encoded digest: %s"
msgstr "base64 kódolási kivonat: %s"

#: keystone/common/wsgi.py:73
#, python-format
msgid "Starting %(arg0)s on %(host)s:%(port)s"
msgstr "%(arg0)s indítása ezen: %(host)s:%(port)s"

#: keystone/common/wsgi.py:196
#, python-format
msgid "arg_dict: %s"
msgstr "arg_dict: %s"

#: keystone/common/wsgi.py:217
#, python-format
msgid "Authorization failed. %s from %s"
msgstr "Hitelesítés sikertelen. %s innen: %s"

#: keystone/common/wsgi.py:430
msgid "The resource could not be found."
msgstr "Az erőforrás nem található."

#: keystone/common/ldap/core.py:171
#, python-format
msgid "Duplicate name, %s."
msgstr "Többszörös név: %s."

#: keystone/common/ldap/core.py:181
#, python-format
msgid "Duplicate ID, %s."
msgstr "Többszörös azonosító: %s."

#: keystone/common/ldap/core.py:186
#, python-format
msgid "LDAP backend does not allow %s create"
msgstr "Az LDAP háttérprogram nem engedélyezi %s létrehozását"

#: keystone/common/ldap/core.py:293
#, python-format
msgid "LDAP backend does not allow %s update"
msgstr "Az LDAP háttérprogram nem engedélyezi %s frissítését"

#: keystone/common/ldap/core.py:321
#, python-format
msgid "LDAP backend does not allow %s delete"
msgstr "Az LDAP háttérprogram nem engedélyezi %s törlését"

#: keystone/common/ldap/fakeldap.py:148
#, python-format
msgid "FakeLdap initialize url=%s"
msgstr "FakeLdap előkészítési url=%s"

#: keystone/common/ldap/fakeldap.py:158
#, python-format
msgid "FakeLdap bind dn=%s"
msgstr "FakeLdap bind dn=%s"

#: keystone/common/ldap/fakeldap.py:165
#, python-format
msgid "FakeLdap bind fail: dn=%s not found"
msgstr "FakeLdap bind hiba: dn=%s nem található"

#: keystone/common/ldap/fakeldap.py:172
#, python-format
msgid "FakeLdap bind fail: password for dn=%s not found"
msgstr "FakeLdap bind hiba: a jelszó nem található ehhez: dn=%s"

#: keystone/common/ldap/fakeldap.py:177
#, python-format
msgid "FakeLdap bind fail: password for dn=%s does not match"
msgstr "FakeLdap bind hiba: a jelszó nem egyezik ehhez: dn=%s"

#: keystone/common/ldap/fakeldap.py:192
#, python-format
msgid "FakeLdap add item: dn=%s, attrs=%s"
msgstr "FakeLdap elem hozzáadása: dn=%s, attrs=%s"

#: keystone/common/ldap/fakeldap.py:194
#, python-format
msgid "FakeLdap add item failed: dn=%s is already in store."
msgstr "FakeLdap elem hozzáadása sikertelen: dn=%s már a tárolóban van."

#: keystone/common/ldap/fakeldap.py:208 keystone/common/ldap/fakeldap.py:222
#, python-format
msgid "FakeLdap delete item: dn=%s"
msgstr "FakeLdap elem törlése: dn=%s"

#: keystone/common/ldap/fakeldap.py:212 keystone/common/ldap/fakeldap.py:226
#, python-format
msgid "FakeLdap delete item failed: dn=%s not found."
msgstr "FakeLdap elem törlése sikertelen: dn=%s nem található."

#: keystone/common/ldap/fakeldap.py:241
#, python-format
msgid "FakeLdap modify item: dn=%s attrs=%s"
msgstr "FakeLdap elem módosítása: dn=%s attrs=%s"

#: keystone/common/ldap/fakeldap.py:245
#, python-format
msgid "FakeLdap modify item failed: dn=%s not found."
msgstr "FakeLdap elem módosítása sikertelen: dn=%s nem található."

#: keystone/common/ldap/fakeldap.py:262
#, python-format
msgid "FakeLdap modify item failed: item has no attribute \"%s\" to delete"
msgstr "FakeLdap elem módosítása sikertelen: az elemnek nincs törölhető „%s” attribútuma"

#: keystone/common/ldap/fakeldap.py:273
#, python-format
msgid ""
"FakeLdap modify item failed: item has no attribute \"%s\" with value \"%s\" "
"to delete"
msgstr "FakeLdap elem módosítása sikertelen: az elemnek nincs törölhető „%s” attribútuma „%s” értékkel"

#: keystone/common/ldap/fakeldap.py:278
#, python-format
msgid "FakeLdap modify item failed: unknown command %s"
msgstr "FakeLdap elem módosítása sikertelen: ismeretlen parancs: %s"

#: keystone/common/ldap/fakeldap.py:280
#, python-format
msgid "modify_s action %s not implemented"
msgstr "modify_s %s művelet nincs megvalósítva"

#: keystone/common/ldap/fakeldap.py:298
#, python-format
msgid "FakeLdap search at dn=%s scope=%s query=%s"
msgstr "FakeLdap keresés itt: dn=%s hatókör=%s lekérdezés=%s"

#: keystone/common/ldap/fakeldap.py:304
msgid "FakeLdap search fail: dn not found for SCOPE_BASE"
msgstr "FakeLdap keresés sikertelen: a dn nem található a SCOPE_BASE-hez"

#: keystone/common/ldap/fakeldap.py:318
#, python-format
msgid "Search scope %s not implemented."
msgstr "A(z) %s keresési hatókör nincs megvalósítva."

#: keystone/common/sql/core.py:163
#, python-format
msgid "Got mysql server has gone away: %s"
msgstr "A kapott MySQL szerver eltűnt: %s"

#: keystone/common/sql/legacy.py:174
#, python-format
msgid "Cannot migrate EC2 credential: %s"
msgstr "Nem migrálhatók az EC2 hitelesítési adatok: %s"

#: keystone/common/sql/nova.py:58
#, python-format
msgid "Create tenant %s"
msgstr "%s bérlő létrehozása"

#: keystone/common/sql/nova.py:74
#, python-format
msgid "Create user %s"
msgstr "%s felhasználó létrehozása"

#: keystone/common/sql/nova.py:83
#, python-format
msgid "Add user %s to tenant %s"
msgstr "%s felhasználó hozzáadása %s bérlőhöz"

#: keystone/common/sql/nova.py:91
#, python-format
msgid "Ignoring existing role %s"
msgstr "Meglévő %s szerep figyelmen kívül hagyása"

#: keystone/common/sql/nova.py:98
#, python-format
msgid "Create role %s"
msgstr "%s szerep létrehozása"

#: keystone/common/sql/nova.py:108
#, python-format
msgid "Assign role %s to user %s on tenant %s"
msgstr "%s szerep hozzárendelése %s felhasználóhoz %s bérlőben"

#: keystone/common/sql/nova.py:123
#, python-format
msgid "Creating ec2 cred for user %s and tenant %s"
msgstr "EC2 hitelesítési adatok létrehozása %s felhasználóhoz és %s bérlőhöz"

#: keystone/policy/backends/rules.py:95
#, python-format
msgid "enforce %s: %s"
msgstr "%s kikényszerítése: %s"

#: keystone/token/backends/memcache.py:81
msgid "Unable to add token user list."
msgstr "Nem vehető fel a token felhasználólistája."

#: keystone/token/backends/memcache.py:91
msgid "Unable to add token to revocation list."
msgstr "A token nem adható a visszavonási listához."
